proj/src/main/webapp/script.js

```diff
@@ -23,7 +21,7 @@ if(btnLogin){
         const auth = firebase.auth();
 
         const promise = auth.signInWithEmailAndPassword(emailVal, passVal).then(function(user){
-            window.location.replace("static/chat.html");
+            window.location.replace("chat.html");
```

Comment:
how come it doesn't need to go to the static folder to get the chat.html?

Reply:
because both welcome.html and signup.html are already in the static folder

Reply:
gotcha